resources/conf/config.toml.template

```diff
@@ -481,6 +481,8 @@ enabled = true
 [analytics]
   # Enable/Disable Analytics in Choreo Connect
   enabled = false
+  # Enables elk analytics, if the type remains unassigned, choreo analytics will enabled
+  type = "elk"
```

Comment:
Should not we include "" as the default value and "elk" in the comment. (Because normally default state of configuration is present in config.toml.template)

Reply:
Added "Default" as the default value as discussed. Kept type = "ELK" as a comment